opentelemetry/proto/collector/entities/v1/entities_service.proto

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opentelemetry.proto.collector.entities.v1;
+
+import "opentelemetry/proto/entities/v1/entities.proto";
+
+option csharp_namespace = "OpenTelemetry.Proto.Collector.Entities.V1";
+option java_multiple_files = true;
+option java_package = "io.opentelemetry.proto.collector.entities.v1";
+option java_outer_classname = "EntitiesServiceProto";
+option go_package = "go.opentelemetry.io/proto/otlp/collector/entities/v1";
+
+// Service that can be used to push entities between one Application instrumented with
+// OpenTelemetry and an collector, or between an collector and a central collector (in this
+// case entities are sent/received to/from multiple Applications).
+service EntitiesService {
+  // For performance reasons, it is recommended to keep this RPC
+  // alive for the entire life of the application.
+  rpc Export(ExportEntitiesServiceRequest) returns (ExportEntitiesServiceResponse) {}
+}
+
+message ExportEntitiesServiceRequest {
+  // An array of ResourceEntities.
+  repeated opentelemetry.proto.entities.v1.ScopeEntities scope_entities = 1;
+}
+
+message ExportEntitiesServiceResponse {
+  // The details of a partially successful export request.
+  //
+  // If the request is only partially accepted
+  // (i.e. when the server accepts only parts of the data and rejects the rest)
+  // the server MUST initialize the `partial_success` field and MUST
+  // set the `rejected_<signal>` with the number of items it rejected.
+  //
+  // Servers MAY also make use of the `partial_success` field to convey
+  // warnings/suggestions to senders even when the request was fully accepted.
+  // In such cases, the `rejected_<signal>` MUST have a value of `0` and
+  // the `error_message` MUST be non-empty.
+  //
+  // A `partial_success` message with an empty value (rejected_<signal> = 0 and
+  // `error_message` = "") is equivalent to it not being set/present. Senders
+  // SHOULD interpret it the same way as in the full success case.
+  ExportEntitiesPartialSuccess partial_success = 1;
+}
+
+message ExportEntitiesPartialSuccess {
+  // The number of rejected log records.
```

Comment:
Is this supposed to say "log records"?

Reply:
Good catch. Copy/paste error. Fixed now.